--- yaboot-1.3.14/Makefile.orig	2007-08-17 16:29:26.000000000 +0200
+++ yaboot-1.3.14/Makefile	2008-09-16 20:22:34.000000000 +0200
@@ -7,8 +7,8 @@
 DEBUG = 0
 # make install vars
 ROOT =
-PREFIX = usr/local
-MANDIR = man
+PREFIX = usr
+MANDIR = share/man
 # command used to get root (needed for tarball creation)
 GETROOT = fakeroot
 
@@ -178,32 +178,30 @@
 	strip --remove-section=.comment --remove-section=.note util/addnote
 
 install: all strip
-	install -d -o root -g root -m 0755 ${ROOT}/etc/
-	install -d -o root -g root -m 0755 ${ROOT}/${PREFIX}/sbin/
-	install -d -o root -g root -m 0755 ${ROOT}/${PREFIX}/lib
-	install -d -o root -g root -m 0755 ${ROOT}/${PREFIX}/lib/yaboot
-	install -d -o root -g root -m 0755 ${ROOT}/${PREFIX}/${MANDIR}/man5/
-	install -d -o root -g root -m 0755 ${ROOT}/${PREFIX}/${MANDIR}/man8/
-	install -o root -g root -m 0644 second/yaboot ${ROOT}/$(PREFIX)/lib/yaboot
-	install -o root -g root -m 0755 util/addnote ${ROOT}/${PREFIX}/lib/yaboot/addnote
-	install -o root -g root -m 0644 first/ofboot ${ROOT}/${PREFIX}/lib/yaboot/ofboot
-	install -o root -g root -m 0755 ybin/ofpath ${ROOT}/${PREFIX}/sbin/ofpath
-	install -o root -g root -m 0755 ybin/ybin ${ROOT}/${PREFIX}/sbin/ybin
-	install -o root -g root -m 0755 ybin/yabootconfig ${ROOT}/${PREFIX}/sbin/yabootconfig
+	install -d -m 0755 ${ROOT}/etc/
+	install -d -m 0755 ${ROOT}/${PREFIX}/sbin/
+	install -d -m 0755 ${ROOT}/${PREFIX}/lib
+	install -d -m 0755 ${ROOT}/${PREFIX}/lib/yaboot
+	install -d -m 0755 ${ROOT}/${PREFIX}/${MANDIR}/man5/
+	install -d -m 0755 ${ROOT}/${PREFIX}/${MANDIR}/man8/
+	install -m 0644 second/yaboot ${ROOT}/$(PREFIX)/lib/yaboot
+	install -m 0755 util/addnote ${ROOT}/${PREFIX}/lib/yaboot/addnote
+	install -m 0644 first/ofboot ${ROOT}/${PREFIX}/lib/yaboot/ofboot
+	install -m 0755 ybin/ofpath ${ROOT}/${PREFIX}/sbin/ofpath
+	install -m 0755 ybin/ybin ${ROOT}/${PREFIX}/sbin/ybin
+	install -m 0755 ybin/yabootconfig ${ROOT}/${PREFIX}/sbin/yabootconfig
 	rm -f ${ROOT}/${PREFIX}/sbin/mkofboot
 	ln -s ybin ${ROOT}/${PREFIX}/sbin/mkofboot
-	@gzip -9 man/*.[58]
-	install -o root -g root -m 0644 man/bootstrap.8.gz ${ROOT}/${PREFIX}/${MANDIR}/man8/bootstrap.8.gz
-	install -o root -g root -m 0644 man/mkofboot.8.gz ${ROOT}/${PREFIX}/${MANDIR}/man8/mkofboot.8.gz
-	install -o root -g root -m 0644 man/ofpath.8.gz ${ROOT}/${PREFIX}/${MANDIR}/man8/ofpath.8.gz
-	install -o root -g root -m 0644 man/yaboot.8.gz ${ROOT}/${PREFIX}/${MANDIR}/man8/yaboot.8.gz
-	install -o root -g root -m 0644 man/yabootconfig.8.gz ${ROOT}/${PREFIX}/${MANDIR}/man8/yabootconfig.8.gz
-	install -o root -g root -m 0644 man/ybin.8.gz ${ROOT}/${PREFIX}/${MANDIR}/man8/ybin.8.gz
-	install -o root -g root -m 0644 man/yaboot.conf.5.gz ${ROOT}/${PREFIX}/${MANDIR}/man5/yaboot.conf.5.gz
-	@gunzip man/*.gz
+	install -m 0644 man/bootstrap.8 ${ROOT}/${PREFIX}/${MANDIR}/man8/bootstrap.8
+	install -m 0644 man/mkofboot.8 ${ROOT}/${PREFIX}/${MANDIR}/man8/mkofboot.8
+	install -m 0644 man/ofpath.8 ${ROOT}/${PREFIX}/${MANDIR}/man8/ofpath.8
+	install -m 0644 man/yaboot.8 ${ROOT}/${PREFIX}/${MANDIR}/man8/yaboot.8
+	install -m 0644 man/yabootconfig.8 ${ROOT}/${PREFIX}/${MANDIR}/man8/yabootconfig.8
+	install -m 0644 man/ybin.8 ${ROOT}/${PREFIX}/${MANDIR}/man8/ybin.8
+	install -m 0644 man/yaboot.conf.5 ${ROOT}/${PREFIX}/${MANDIR}/man5/yaboot.conf.5
 	@if [ ! -e ${ROOT}/etc/yaboot.conf ] ; then						\
-		echo "install -o root -g root -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf"; \
-		install -o root -g root -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf;	\
+		echo "install -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf"; \
+		install -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf;	\
 	 else											\
 		echo "/etc/yaboot.conf already exists, leaving it alone";			\
 	 fi
